pinot-controller/src/test/java/org/apache/pinot/controller/helix/SegmentStatusCheckerTest.java

```diff
@@ -340,12 +340,15 @@ public void missingEVPartitionPushTest() {
     idealState.setPartitionState("myTable_0", "pinot1", "ONLINE");
     idealState.setPartitionState("myTable_1", "pinot1", "ONLINE");
     idealState.setPartitionState("myTable_1", "pinot2", "ONLINE");
+    idealState.setPartitionState("myTable_2", "pinot1", "ONLINE");
+    idealState.setPartitionState("myTable_2", "pinot2", "ONLINE");
     idealState.setReplicas("2");
     idealState.setRebalanceMode(IdealState.RebalanceMode.CUSTOMIZED);
 
     ExternalView externalView = new ExternalView(offlineTableName);
     externalView.setState("myTable_1", "pinot1", "ONLINE");
     externalView.setState("myTable_1", "pinot2", "ONLINE");
+    externalView.setState("myTable_2", "pinot1", "ONLINE");
```

Comment:
It'd be good to add some description here for better understanding of the intention of the test you added.

Reply:
Updated